Library/Formula/ola.rb

```diff
@@ -20,14 +21,15 @@ class Ola < Formula
 
   option :universal
   option "with-ftdi", "Install FTDI USB plugin for OLA."
+  option "with-rdm-tests", "Install RDM Tests for OLA."
 
   depends_on "pkg-config" => :build
   depends_on "cppunit"
```

Comment:
I'd like to add a TODO if there's an issue covering adding a :test dependency flag.